ansible/roles/gelato-ha-installer/tasks/main.yml

```diff
@@ -23,6 +23,17 @@
   with_items:
     - "{{ gelato_work_dir }}"
 
+- name: get the IPs of API Service.
+  shell: kubectl get nodes k8smaster -o wide --no-headers | awk '{print $6}'
```

Comment:
IMO, ```k8smaster``` may not the node name always

Reply:
In what situation , this is a prerequisite

Reply:
It would be mentioned in the documentation

Reply:
ok .